src/GraphNodeManagerViewExtension/GraphNodeManagerView.xaml

```diff
@@ -54,31 +54,39 @@
                     </ItemsControl.ItemsPanel>
                 </ItemsControl>
             </Grid>
-
             <Image x:Name="ExportImage"
-                   DockPanel.Dock="Right"
-                   Width="16px"
-                   Height="16px"
-                   Margin="15 0 15 15"
-                   HorizontalAlignment="Right"
-                   VerticalAlignment="Center"
-                   Source="/GraphNodeManagerViewExtension;component/Images/Export.png"
-                   Stretch="UniformToFill"
-                   MouseUp="ExportImage_OnMouseUp"
-                   Cursor="Hand">
+                       DockPanel.Dock="Right" 
```

Comment:
Same, remove additional spacing to align correctly.

Reply:
Can you check if you see it now, @reddyashish? The whole section has been restructured.